src/employeeProfiles.js

```diff
@@ -167,15 +169,37 @@ export const EmployeeProfile = () => {
             <TextField label="" source="personalEmail" />
           </SimpleShowLayout>
         </Grid>
-        {/*
-      <Grid item>
-        <SimpleShowLayout divider={<Divider flexItem />}>
-          <TextField label="Current assigment" source="" />
-          <DateField label="Hired Date" source="" />
-        </SimpleShowLayout>
-      </Grid>
-      Hidden empty fields until developed
-      */}
+        <Grid item>
+          <FunctionField
+            label=""
+            render={(record) => (
+              <SimpleShowLayout divider={<Divider flexItem />}>
+                {!record.startDate && (
+                  <span>This employee does not have any contracts</span>
+                )}
+                {record.startDate && (
+                  <TextField
+                    label="Start Date"
+                    source="startDate"
+                    record={record}
+                  />
+                )}
+                {record.startDate && !record.endDate && (
+                  <span>
+                    This employee does not have an end date for the contract
```

Comment:
minor: i would change to:  <span> - </span>, what do you think guys?

Reply:
I think it's a good idea, I uploaded it there